metaflow/plugins/kfp/tests/test_kfp_s3_sensor.py

```diff
@@ -24,6 +30,68 @@ def identity_formatter(path: str, flow_parameters: dict) -> str:
     return path
 
 
+@mock_s3
+@pytest.mark.parametrize(
+    "upload_bucket, upload_key, upload_path, processed_path, flow_parameters_json, os_vars",
+    [
+        (
+            "sample_bucket",
+            "sample_prefix/sample_file.txt",
+            "s3://sample_bucket/sample_prefix/sample_file.txt",
+            "s3://sample_bucket/sample_prefix/sample_file.txt",
+            '{"key": "value"}',
+            False,
+        ),
+        (
+            "sample_bucket",
+            "sample_prefix/date=07-02-2021/sample.txt",
+            "s3://sample_bucket/sample_prefix/date={date}/sample.txt",
+            "s3://sample_bucket/sample_prefix/date=07-02-2021/sample.txt",
+            '{"date": "07-02-2021"}',
+            False,
+        ),
+        (
+            "sample_bucket",
+            "sample_prefix/date=08-03-2022/sample.txt",
+            "s3://sample_bucket/sample_prefix/date=$DATE/sample.txt",
+            "s3://sample_bucket/sample_prefix/date=08-03-2022/sample.txt",
+            '{"key": "value"}',
+            True,
+        ),
+    ],
+)
+def test_wait_for_s3_path_no_exception(
```

Comment:
Instead of calling it no exception, just call it `test_wait_for_s3_path` or `test_wait_for_s3_path_vanilla` rather than what it is not